Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -254,6 +254,21 @@ - (void)authorizationController:(ASAuthorizationController *)controller didCompl
     return requests;
 }
 
+void WebAuthenticatorCoordinatorProxy::pauseConditionalAssertion()
+{
+    m_paused = true;
+    if (m_isConditionalAssertion)
+        [m_controller cancel];
+}
+
+void WebAuthenticatorCoordinatorProxy::unpauseConditionalAssertion()
+{
+    if (m_paused && m_controller && m_isConditionalAssertion)
```

Comment:
I think we should early return if `m_paused` is already false?

Reply:
Early return seems good, though the way this code is structured there's no real advantage (aside from avoiding re-assigning `m_paused` to `true`.